packages/db/package.json

```diff
@@ -32,6 +32,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@esbuild/linux-x64": "^0.24.0",
```

Comment:
I needed this to build on linux, and I'm not sure if someone else has changed prisma yet. This may have to be removed before merge,

Reply:
revert this please and tag people when you see errors you dont understand

Reply:
reverted - I bet I messed up the lock file though.

Reply:
reverted lock file, this one should be fine now. Except if the error comes back, which if it does will post in contributors.